notebooks/211-speech-to-text/211-speech-to-text.ipynb

```diff
@@ -78,8 +78,7 @@
     "import librosa.display\n",
```

Comment:
<p>probably, you need to install openvino here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
done